.github/templates/main.yml

```diff
@@ -74,10 +74,12 @@ steps:
 #@ end
 
 #@ def buildPackages():
+#@ finalList = []
 #@ for pkgName in nugetPackages:
-  - name: #@ "Build nuget package " + pkgName
-    _: #@ template.replace(msbuild("Realm/" + pkgName, target="Pack", PackageOutputPath="${{ github.workspace }}/Realm/packages", VersionSuffix="${{ steps.set-version-suffix.outputs.build_suffix }}"))
+#@  finalList.append(msbuild("Realm/" + pkgName, target="Pack", PackageOutputPath="${{ github.workspace }}/Realm/packages", VersionSuffix="${{ steps.set-version-suffix.outputs.build_suffix }}"))
```

Comment:
Any benefit of aggregating that in an array vs just returning the nodes as we iterate?

Reply:
At line 307 I use `template.replace` since I need to replace with a list of objects. `template.replace` wants something returned from the function, that's why I return an array. The modification was made because you expressed preference to use `msbuild` as is without using `temple.replace` for every invocation. This made the function `msbuild` not return anymore an array and so I had to rebuild the array in `buildPackage`.